doc/dnf5.conf.5.rst

```diff
@@ -1261,6 +1261,43 @@ Files
     Filenames may contain only alphanumeric characters and underscores and be in lowercase.
     Variables are also read from ``/etc/yum/vars`` for YUM compatibility reasons.
 
+
+Directories
+===========
+
+Directories are listed in the opposite order in which they are
+processed meaning that the first directory in each list
+has the highest priority in its category.
+
+Repository Directories
```

Comment:
The repository configuration directories are processed in the following order: `/etc/yum.repos.d/`, `/etc/distro.repos.d/`, `/usr/share/dnf5/repos.d`.

The important thing is that, unlike overrides that modify an existing repository configuration, the configuration in these directories defines new repositories. And we cannot define multiple repositories with the same ID. If multiple repositories with the same ID are defined only the first one will be loaded. Loading the next one will log an error.

Reply:
updated the docs